components/AvalancheCenterLogo.tsx

```diff
@@ -34,57 +31,62 @@ export const AvalancheCenterLogo: React.FunctionComponent<AvalancheCenterLogoPro
     ['WAC']: Image.resolveAssetSource(require('../assets/logos/WAC.png')),
   };
 
+  const {data: uri} = useCachedImageURI(source[avalancheCenterId].uri);
+  if (!uri) {
+    return <ActivityIndicator style={style} />;
+  }
+
   const images: Record<string, {(s: ImageStyle): ReactElement}> = {
     ['BTAC']: (s: ImageStyle) => {
-      return <Image style={s} source={require('../assets/logos/BTAC.png')} />;
+      return <Image style={s} source={{uri: uri, cache: 'force-cache'}} />;
```

Comment:
I am still super confused about this. If we're requiring something from the app bundle, why do we need to cache it? It's already in storage!

Reply:
also this is referring to the http cache which I assume would not be in play for bundle assets?

Reply:
I addressed this more in the commit message, but the long-and-short of it is that in developer mode, Expo will serve assets via the local network even if those assets would otherwise be bundled into the binary in production. So, this is "just" to support local development, but you'll see the hook no-ops for `file://` URIs in the first place and I think it's very nice to have this working locally to make sure we actually work :)

Reply:
hmm ok. I would add a comment to that effect in the code because it's non-obvious and the code looks strange to someone coming in cold.

I would also try to skip calling `ImageCache.prefetch` unless we're in `__DEV__` mode since it just wastes cache space.

Reply:
> I would also try to skip calling ImageCache.prefetch unless we're in __DEV__ mode since it just wastes cache space.

If you call these with a `file://` URL, they no-op.

Reply:
Added a comment to the hook.